cmd/api/src/utils/validation/min_max_validator.go

```
@@ -0,0 +1,78 @@
+type MinMaxValidator struct {
+	min, max 	   string
+	minVal, maxVal int
+}
+
+func NewMinMaxValidator(params map[string]string) Validator {
+	validator := MinMaxValidator{}
+
+	if minVal, ok := params["min"]; ok {
+		validator.min = params["min"]
+		if val, err := strconv.Atoi(minVal); err != nil {
+			slog.Warn("NewMinMaxValidator invalid min limit provided", slog.String("min value", minVal))
```

golangci-lint (2.11.4)

[major] keys should be written in snake_case

(sloglint)